### ISSUE TITLE
catalog data already in DFHG

### ISSUE THREAD

**Issue by cwulfman** [PerseusDL/catalog_data#142]
Both projects have sets of authorized names; e.g.,

urn:cite:perseus:author.1488.1 and urn:cite:lofts:fhg.1.xanthus  both appear to be identifiers for the same person. Is there a mapping somewhere?

Also, I see from the http://www.dfhg-project.org/Fragmentary-Authors-Catalog/ that the mapping from metadata about fragments to pages in a digitized copy of Muller already exists. I can't find that data on the DFHG site, though. Do you know where it is?

**Comment by AlisonBabeu** [comment]
I don't know in all honesty @cwulfman if there is a mapping somewhere between the lofts CITE URNs and the Perseus Catalog URNs.  Also in terms of a mapping from metadata to authors to pages, it looks like you can find that in the individual author entries like for Critias (http://www.dfhg-project.org/Fragmentary-Authors-Catalog/index.php?what%5B%5D=author%7CCritias+Atheniensis&onoffswitch=on) in the fifth column of data, "pages", it links directly to a copy within the Internet Archive. 
Its pretty much the same type of data in the expression level pages in the Perseus Catalog. I'm not sure if there is a master mapping table somewhere.

And if you want background date regarding Mueller-Jacoby, what type of information were you needing?

**Comment by cwulfman** [comment]
That php call is drawing on data in some database somewhere; I was wondering if I could have access to that data in a spreadsheet or something.

A mapping between those identifiers seems pretty important, don't you think? I hadn't realized how distinct these projects are. It would be terrific if we could all be using the same identifiers to talk about the same things (indeed, not much use doing what we're doing here if we don't!) The OWL ontology has a predicate, owl:sameAs, the express equivalence, but that's liable to get messy and difficult to maintain.

I'm not telling you anything you don't already know, though!

**Comment by AlisonBabeu** [comment]
Perhaps it is time to bring Monica into this conversation then? She might
be able to tell us where such a mapping lives!

**Comment by cwulfman** [comment]
Yes, I think so. Especially as she is one of the big stakeholders for the catalog.
